app/controllers/friendships_controller.rb

```diff
@@ -5,6 +5,7 @@ class FriendshipsController < ApplicationController
   before_action :set_user
```

Comment:
You'll need to test this on your end to see why it is not working. Below are steps I took to test and it broke:
Log in -> LeagueSpec Social (navbar) -> All Friends. I got the error "uninitialized constant FriendshipsController::LolConnections"
Another step I tried to test with: 
From the search bar, search for a user that you know you are already friends with -> Returns the error: "NameError in Search#search"

If any of these tests work for you, then let me know.